tests/server/test_server_routes_populate_queues.py

```diff
@@ -804,21 +806,65 @@ def test_start_recording_command__populates_queue__with_correctly_parsed_set_of_
         "metadata_to_copy_onto_main_file_attributes"
     ][PLATE_BARCODE_UUID]
     response = test_client.get(
-        f"/start_recording?barcode={expected_barcode}&active_well_indices=0,5,8&is_hardware_test_recording=False"
+        f"/start_recording?barcode={expected_barcode}&active_well_indices=0,5,8"
     )
     assert response.status_code == 200
 
     comm_queue = (
         test_process_manager.queue_container().get_communication_queue_from_server_to_main()
     )
-    confirm_queue_is_eventually_of_size(
-        comm_queue, 1
-    )  # assert is_queue_eventually_not_empty(comm_queue) is True
+    confirm_queue_is_eventually_of_size(comm_queue, 1)
     communication = comm_queue.get(timeout=QUEUE_CHECK_TIMEOUT_SECONDS)
     assert communication["command"] == "start_recording"
     assert set(communication["active_well_indices"]) == set([0, 8, 5])
 
 
+@pytest.mark.parametrize(
+    "scanned_barcode,user_entered_barcode,expected_result,test_description",
+    [
+        (
+            "MA200440001",
```

Comment:
do you think it's relevant to test an empty string `""`?  I don't recall what ok_comm passes back to processMonitor if the barcode fails to scan

Reply:
It does pass an empty string when the scanner fails. I can add that test in the next PR fairly quickly